awslambdaric/bootstrap.py

```diff
@@ -333,20 +385,27 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_tb):
         self.file.close()
 
-    def log(self, msg):
+    def set_log_level(self, frame_type, log_level):
```

Comment:
Can you inline this function? I think if we expose that name, customer will think that they can change the log level.